charts/flyte-core/values-eks.yaml

```diff
@@ -50,6 +50,37 @@ flyteadmin:
               app.kubernetes.io/name: flyteadmin
           topologyKey: kubernetes.io/hostname
 
+#
+# FLYTESCHEDULER
+#
+
+flytescheduler:
```

Comment:
For EKS probably we dont want to bring up this and use the AWS scheduler for it which is currently used.
Since all the current schedules for EKS deployments would be available there and we need not bring this up in that environment

Reply:
ok